_posts/hannahlwang/2016-02-15-hannah-midsemester-reflection.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author: hannahlwang
+title: "Hannah's Midsemester Reflection"
+---
+
+I think the greatest challenge for me this semester has been assessing what I don’t know, and figuring out how I can fill or improve those knowledge gaps. The book exercises have provided great hands-on examples of the topics covered, but I have learned to take the more freeform assignments as chances to improve my skill set and try something that I might not know how to do. 
+
+<h3>Clicky Turtle</h3>
+
+<iframe src="https://trinket.io/embed/python/a89c1fca50" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+My clicky turtle was one of the exercises where I felt like I did this successfully. Prior to this assignment, I had done a lot of work with circles in my other programs. I had made Tina the Turtle draw a modernist clock, a cake with different colors, and differently sized balloons, all of which mostly consisted of circles. Although these drawings all looked nice, I knew that I would have to try some other shapes in order to expand my skill set. For this clicky turtle assignment, I had Sally the Turtle set up a lily pond with lily pads - these were circles with slices taken out of them, which required me to do more thinking about angles. The real challenge here was getting Tina the Turtle to draw the star polygons I had seen in my classmates’ work. The problem-solving technique I used here was Wikipedia.
+
+In retrospect, I really wish I had gone with my original idea for this assignment - creating ripples in the water where the user clicked. This would have been incredibly challenging, but I would have learned how to use clear, and it would have required more difficult loops. Maybe I can use this as a challenge for a future assignment. 
```

Comment:
This could definitely be a part of your drawing app.  You have the tools you need now with circles, time.sleep and turtle.clear()